### ISSUE TITLE
Failed to build android

### ISSUE THREAD

**Issue by Tryibion** [FlaxEngine/FlaxEngine#2052]
**Issue description:**
I get this error when tiring to build for Android
```
[ 00:02:55.862 ]: [Info] Android, ARM64, C:\Program Files/dotnet/packs/Microsoft.NETCore.App.Runtime.Mono.android-arm64/8.0.0/runtimes/android-arm64/native
[ 00:02:55.887 ]: [Info] Using .Net Runtime Host at C:\Program Files/dotnet/packs/Microsoft.NETCore.App.Runtime.Mono.android-arm64/8.0.0/runtimes/android-arm64/native
[ 00:02:55.890 ]: [Error] Failed to copy .Net runtime data files.
[ 00:02:55.890 ]: [Error] Game building failed!
```

**Flax version:**
1.7.1

**Comment by fabgames5** [comment]
I Second this, as I've yet to be able to make a Android build successfully...

**Comment by mafiesto4** [comment]
It might be a bug in dotnet files deployment scripts when using new .NET 8. We gonna need to address that.

**Comment by Tryibion** [comment]
Looks like if I replace all of the .net7 references with .net8 in the deploy data step cpp file then it works fine. not sure why it is reliant on .net 8 though.

**Comment by mafiesto4** [comment]
If you use prebuilt engine from Launcher then it was compiled with .NET 7 so using newer dotnet might be a problem on cooked game (eg. Android). We could improve this, also starting with `1.8` we will use dotnet 8 as min version.

**Comment by mafiesto4** [comment]
Fixed on `1.8` branch.
